packages/sbom-tools/test/helpers.ts

```diff
@@ -0,0 +1,28 @@
+import fs from 'fs';
+import path from 'path';
+import os from 'os';
+import { rimrafSync } from 'rimraf';
+
+export const importFixture = async (file: string) =>
+  (await import(`./fixtures/${file}`)).default;
+
+export const toCleanup: string[] = [];
```

Comment:
Just a note, but it’s probably not the ideal way of doing things to have a global list here. In the tests for the products, we typically generate temporary directories in a `before` or `beforeEach` hook and then do the cleanup in a corresponding `after` or `afterEach` hook.

Also, these functions here seem to be used mostly from async functions or functions that could be made async functions fairly easily – maybe it’s a good idea to use normal fs functions rather than the sync ones? I know it doesn’t make a real difference for tests, but it’s a good practice in general and doesn’t really add extra code.

Reply:
We can improve this yeah, i thought it is fine in this context, those tests should not run in parallel, but yeah, we can code this properly

Reply:
It is fine, and again, this is optional, it’s just not really matching what our tests usually do, and the way that our tests usually do it does have its advantages. But yeah, not a big deal unless we start copying this kind of code around

Reply:
yeah, no is a very good point, let's do it in a way that is up to our standard, is a quick thing to change it

Reply:
Updated, let me know if that's ok